docs/terraform.md

```diff
@@ -111,8 +111,8 @@
 | stage | Stage (e.g. `prod`, `dev`, `staging`) | string | `` | no |
 | tags | Additional tags (_e.g._ { BusinessUnit : ABC }) | map(string) | `<map>` | no |
 | task_cpu | The number of CPU units used by the task. If unspecified, it will default to `container_cpu`. If using `FARGATE` launch type `task_cpu` must match supported memory values (https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_size) | number | `null` | no |
+| task_ignore_changes | Ignore changes (like environment variables) to the ECS task definition | bool | `true` | no |
 | task_memory | The amount of memory (in MiB) used by the task. If unspecified, it will default to `container_memory`. If using Fargate launch type `task_memory` must match supported cpu value (https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_size) | number | `null` | no |
-| ulimits | The ulimits to configure for the container. This is a list of maps. Each map should contain "name", "softLimit" and "hardLimit" | object | `<list>` | no |
```

Comment:
why `ulimits` was removed from the generated docs?
Can you please run these commands again:

```
make init
make readme/deps
make readme
```

Reply:
This is simply what happened when I ran this. I re-ran and confirmed ulimits was present, so this should be solved.